R/fb_player_scouting.R

```diff
@@ -21,25 +22,27 @@
 #'
 #' # to filter for the last 365 days:
 #' fb_player_scouting_report(player_url = "https://fbref.com/en/players/d70ce98e/Lionel-Messi",
-#' pos_versus = "primary") %>% dplyr::filter(scouting_period == "Last 365 Days")
+#' pos_versus = "primary", league_comp_name = "Last 365 Days Men's Big 5 Leagues, UCL, UEL")
```

Comment:
Defaulting this to `NULL`. Should users want to get reports for specific leagues/comps, they can